tests/ChannelMessagesTest.php

```diff
@@ -269,7 +269,7 @@ public function testPublishConnectionKey() {
         $msg = new Message();
         $msg->name = 'delegatedMsg';
         $msg->data = 'test payload';
-        $msg->connectionKey = 'fake!realtime_key';
+        $msg->connectionKey = 'fake.realtime_key';
```

Comment:
we had the same issue with ably-python
it got fixed in this commit
https://github.com/ably/ably-python/commit/e4078f265ee71fa985db9833aeec77b1ea1d625d

but I understand there is still an issue server side, it should not timeout imho